code/messaging/src/phone/java/com/adobe/marketing/mobile/MessagingPushPayload.java

```diff
@@ -145,7 +145,7 @@ public boolean isAEPPushMessage() {
             Log.error(MessagingConstants.LOG_TAG, "%s - Returning false as remote message data payload is empty", SELF_TAG);
             return false;
         }
-        return "true".equals(data.get(MessagingConstants.PushNotificationPayload.ADB));
+        return data.get(MessagingConstants.PushNotificationPayload.XDM) != null;
```

Comment:
did the payload from ajo change?

Reply:
i believe it has since this function was originally added. there is no longer a `adb_ = true` present in the payload so i am checking for the presence of `_xdm` instead.